app/src/main/java/org/fossasia/pslab/activity/ReportUs.java

```diff
@@ -17,7 +17,7 @@
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_report_us);
```

Comment:
Yes. might have done some copy pasting in the wrong file accidently. I will correct it.